Karl/schema_karl.h

```diff
@@ -226,10 +226,11 @@ CURRENT_STRUCT(up) {
   CURRENT_FIELD_DESCRIPTION(uptime, "The uptime of the service based on the most recent keepalive, human-readable.");
 
   CURRENT_DEFAULT_CONSTRUCTOR(up) {}
-  CURRENT_CONSTRUCTOR(up)(const std::chrono::microseconds start_time_epoch_microseconds,
-                          const std::string& last_keepalive_received,
-                          const std::chrono::microseconds last_keepalive_received_epoch_microseconds,
-                          const std::string& uptime)
+  CURRENT_CONSTRUCTOR(up)
+  (const std::chrono::microseconds start_time_epoch_microseconds,
+   const std::string& last_keepalive_received,
+   const std::chrono::microseconds last_keepalive_received_epoch_microseconds,
+   const std::string& uptime)
```

Comment:
Indeed, it's ugly. How do you feel about https://github.com/dkorolev/Current/commit/9cbf35c17d65bbec8d935ab96ff74cc3c43e8793?

I'm OK keeping the space in `if (...)`, `for (...)`, and `while (...)`, but that would also introduce them into `CURRENT_STRUCT (Foo)` and `CURRENT_CONTSTRUCTOR (Foo)`.